### PR TITLE
Changes the detective's jobslots to 1 roundstart and 2 total

### PR DESCRIPTION
**Pull request by ReinaCoder**
## About The Pull Request

Changes the detective's job slots so there's now only one roundstart, with an additional 1 latejoin slot.
## Why It's Good For The Game
The detective fulfills a specialized role of investigating and solving crime using their brain and their scanner. They lack content, as most of the time this results in being called to one place and walking over to scan it and checking the results. When there's multiple detectives the small amount of work the detective has is now split between 5 people rushing to see who can do it first. This causes what could be a substantial amount of content for a detective in a round divided between 5 people leaving little for all. 

Decreasing the total job slots allows for a better detective experience where it's not a competition as to who will actually have things to do in the round. Two detectives total still allows for buddy cop and for the second detective to know and choose if they want to play a round with another detective.
image
here is my oc comic explaining my logic

## Changelog

:cl:
1 detective roundstart
1 detective latejoin
2 detective slots in total
/:cl:

### CODE REVIEW

**Comment by SGSDHSD-Enterprise** [comment]
fulp station is saved

**Comment by JohnFulpWillard** [comment]
vroom vrrom

**Comment by Pepsilawn** [comment]
i wont have to beat the other detective to death for the gun now

File: fulp_modules/Z_edits/job_edits.dm [hunk 1]
```diff
@@ -11,8 +11,8 @@
 	spawn_positions = 2
 
 /datum/job/detective
-	total_positions = 2
-	spawn_positions = 5
+	total_positions = 1
```
Comment (Pepsilawn) [inline fulp_modules/Z_edits/job_edits.dm:14]:
```suggestion
	total_positions = 2
```

File: fulp_modules/Z_edits/job_edits.dm [hunk 2]
```diff
@@ -11,8 +11,8 @@
...
+	spawn_positions = 2
```
Comment (Pepsilawn) [inline fulp_modules/Z_edits/job_edits.dm:15]:
```suggestion
	spawn_positions = 1
```

**Pull request merged**